tasks/host_volume.yml

```diff
@@ -0,0 +1,8 @@
+- name: create volume {{ item['name'] }}
+  file:
+    path: "{{ item['path'] }}"
+    owner: "{{ nomad_user }}"
```

Comment:
@ducminhle How about `item['owner'] || nomad_user` (allowing optional setting of ownership)?

Reply:
Thanks for your suggest.